app/controllers/state_file/questions/nc_tax_refund_controller.rb

```diff
@@ -0,0 +1,24 @@
+module StateFile
+  module Questions
+    class NcTaxRefundController < QuestionsController
```

Comment:
if these are the same as `TaxRefundController` except for the form key we could have it inherit from that controller and remove the other methods and just have the form_key method in here [dust]
```suggestion
    class NcTaxRefundController < TaxRefundController
```